server/.sequelize.js

```diff
@@ -11,12 +11,17 @@ const dbConfig = {
   password: dbAuth[1],
   host: dbParams.hostname,
   port: dbParams.port,
+  ssl: false,
   dialect: dbParams.protocol.split(':')[0],
   seederStorage: 'sequelize',
   logging: console.log
 };
 
-if (dbParams.hostname !== 'localhost' && dbParams.hostname !== 'fs-intake-postgres') {
+if (
+  dbParams.hostname !== 'localhost' &&
+  dbParams.hostname !== '127.0.0.1' &&
+  dbParams.hostname !== 'fs-intake-postgres'
```

Comment:
is the `fs-intake-postgres` for docker? if so, maybe we should explicitly name that.

Reply:
@lauraGgit yes, that's for docker and it's specified in the readme. We are hesitant to rename because it follows the naming other docker containers (fs-intake-frontend, fs-intake-server).